store/searchfilters.ts

```diff
@@ -27,11 +27,26 @@ export const useSearchFilters = defineStore({
   id: 'searchFilters',
   state: () => {
     return {
+      fixedColumns: [
+        { name: 'Status', key: 'Status' },
+        { name: 'LastModifiedBy', key: 'Modified By' },
+        { name: 'NameRequestNumber', key: 'NR Number' },
+        { name: 'Names', key: 'Names' },
+        { name: 'ApplicantFirstName', key: 'Applicant First Name' },
+        { name: 'ApplicantLastName', key: 'Applicant Last Name' },
+        { name: 'NatureOfBusiness', key: 'Nature Of Business' },
+        { name: 'ConsentRequired', key: 'Consent Required' },
+        { name: 'Priority', key: 'Priority' },
+        { name: 'ClientNotification', key: 'Notified' },
+        { name: 'Submitted', key: 'Submitted' },
+        { name: 'LastUpdate', key: 'Last Update' },
+        { name: 'LastComment', key: 'Last Comment' }
+      ],
```

Comment:
Would it be possible to use the new enum for the key valuese here?

Reply:
Are `name` or `key` used anywhere else? If so, please consider an enum. If not, go ahead and resolve this conversation.